### DISCUSSION TITLE
Some sort of way to easily grab the right dock scrollbars.

### DISCUSSION THREAD

**Discussion by MewPurPur** [godotengine/godot-proposals#6782] [Editor]
On websites, the scrollbar appears right at the rightmost edge.

![image](https://user-images.githubusercontent.com/85438892/235379491-2fd82b5e-99c1-4c5b-994b-b34318d9486d.png)

I am really missing this from Godot, where I must aim more carefully to hit the scrollbar of the right side docks.

![image](https://user-images.githubusercontent.com/85438892/235379416-d9c53d64-7bcf-402b-b61e-fe09db2d0942.png)

I wonder if maybe we could somehow make this work without it somehow looking weird?

**Comment by Calinou** [comment]
Removing the padding in the editor window should do the trick, but this wasn't too well-received when it was last attempted: https://github.com/godotengine/godot/pull/40287

That said, SplitContainer now supports having a draggable area larger than its visual representation, so we can make padding smaller but also uniform between docks.
